lib/msal-browser/src/operatingcontext/BaseOperatingContext.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Logger } from "@azure/msal-common";
+import { BrowserConfiguration } from "../config/Configuration";
+
+/**
+ * Base class for operating context
+ * Operating contexts are contexts in which MSAL.js is being run
```

Comment:
Nit: Should we give examples here (say: i.e. Embedded app or directly on the browser)?

Reply:
I'll add this in a future PR>